build-artifacts/project-template-gradle/build.gradle

```diff
@@ -83,6 +82,48 @@ def computeBuildToolsVersion() {
 }
 project.ext.selectedBuildType = project.hasProperty("release") ? "release" : "debug"
 
+def getAppIdentifier (packageJsonMap) {
+	def appIdentifier = "";
+	if(packageJsonMap){
+		appIdentifier = packageJsonMap.nativescript.id;
+		if (!(appIdentifier instanceof String))
```

Comment:
Does CLI check if appIdentifier object is valid? if not please do some kind of a check here.

Reply:
Yes there is validation for the id property inside CLI:
https://github.com/NativeScript/nativescript-cli/blob/master/lib/services/android-project-service.ts#L474
https://github.com/NativeScript/nativescript-cli/blob/master/lib/services/android-project-service.ts#L95
https://github.com/NativeScript/nativescript-cli/blob/master/lib/commands/run.ts#L40